src/main/java/org/folio/inventory/support/JsonHelper.java

```diff
@@ -28,14 +30,23 @@ public static String getString(JsonObject representation, String propertyName) {
     return null;
   }
 
-  public static void includeIfPresent(
-    JsonObject representation, String propertyName, String value) {
+  public static <T> void includeIfPresent(
+    JsonObject representation, String propertyName, T value) {
 
-    if (representation != null && StringUtils.isNotBlank(propertyName) && value != null) {
+    if (representation != null && isNotBlank(propertyName) && value != null) {
       representation.put(propertyName, value);
     }
   }
 
+  @SuppressWarnings("OptionalUsedAsFieldOrParameterType")
+  public static <T> void includeIfPresent(
```

Comment:
This method seems a bit redundant. If there is a static `PublicationPeriod.toJson(PublicationPeriod publicationPeriod)`, you can avoid adding this one.

Reply:
Make sense, I've removed this method and made the toJson to be static.